docs/Encoder.md

```diff
@@ -1,57 +1,81 @@
 # AMDC Encoder Input
 
-This document describes the design considerations and implementation details for the encoder input subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and each component discussed in detail. Specifications of each component are provided based on the datasheet.
+This document describes the design considerations and implementation details for the encoder input subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and each component is discussed in detail. Specifications of each component are provided based on the datasheet.
+
+*Note that this discussion applies to AMDC REV D hardware only -- prior AMDC revisions used different front-end circuitry.*
+
+# Relevant Versions of AMDC Hardware
+
+AMDC REV D
 
 ## Design Requirements
 
-There are several design requirements while designing encoder subsystem of AMDC:
-1.	To be able to operate with many standard incremental encoders.
-2.	Inner compatibility between chips: differential to single-ended converter chip output should be compatible with level shifter chip input.
-3.	Level shifter reduces signal voltage levels enough so that they can be transmitted to PicoZed.
+There are several design requirements that must be met for the encoder subsystem of AMDC:

```

Comment:
>several design requirements

I think this should say "**two** design requirements"

Reply:
```suggestion
There are two design requirements that must be met for the encoder subsystem of AMDC:

```